Libraries/LibWeb/CSS/Parser/ArbitrarySubstitutionFunctions.cpp

```diff
@@ -198,6 +201,64 @@ static Vector<ComponentValue> replace_an_attr_function(DOM::AbstractElement& ele
     // NB: Step 6 is a lambda defined at the top of the function.
 }
 
+// https://drafts.csswg.org/css-env/#substitute-an-env
+static Vector<ComponentValue> replace_an_env_function(DOM::AbstractElement& element, GuardedSubstitutionContexts& guarded_contexts, ArbitrarySubstitutionFunctionArguments const& arguments)
+{
+    // AD-HOC: env() is not defined as an ASF, (and was defined before the ASF concept was,) but behaves a lot like one.
```

Comment:
typo: "was,) but" -> "was), but"

Reply:
Actually not a typo - I've been doing this wrong my entire life. :sweat_smile: Fixed!